src/components/Button/components/ButtonBase.tsx

```diff
@@ -104,11 +117,51 @@ const getBoxShadow = (props: IStyledBaseButton) => {
   return `0px 0px 2px ${knitui.shades.blue50}`
 }
 
+const getHorizontalPadding = (props: IStyledBaseButton) => {
+  const {
+    customProps: { size },
+  } = props
+  const paddings = {
+    small: 0.7,
+    medium: 1,
```

Comment:
sure both med and large should have same padding?

Reply:
It was actually copied over from ButtonContent, let me see if increased padding is needed

Reply:
nah then it should be as per the design, you can leave it be